bootstrap/pesticide_seed.py

```
@@ -0,0 +1,1243 @@
+    )
+    business_registration_event_name: Mapped[str | None] = mapped_column(
+        Text, nullable=True
+    )
+    address: Mapped[str | None] = mapped_column(Text, nullable=True)
+    raw_row_hash: Mapped[str] = mapped_column(String(64), nullable=False)
+    source_file_name: Mapped[str] = mapped_column(String(255), nullable=False)
+    source_row_index: Mapped[int] = mapped_column(Integer, nullable=False)
+    created_at: Mapped[str] = mapped_column(String(40), nullable=False)
+    updated_at: Mapped[str] = mapped_column(String(40), nullable=False)
+
+    applications: Mapped[list["ProductApplication"]] = relationship(
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)